__tests__/endpoints/articles.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require("supertest");
+const app = require("../../app");
+const db  = require("../../db/connection");
+const seed = require("../../db/seeds/seed");
+const data = require("../../db/data/test-data");
+
+beforeEach(async () => {
+    await seed(data);
+});
+afterAll(async () => {
+    await db.end();
+});
+
+describe("/api/articles/:article_id", () => {
+    describe("GET", () => {
+        describe("if valid article_id", () => {
+            test("200: responds with an article object", () => {
+                return request(app)
+                    .get("/api/articles/1")
+                    .expect(200)
+                    .then((response) => {
+                        expect(response.body.article).toMatchObject({
+                            author: expect.any(String),
```

Comment:
you can see what is in the article so you can expect for exactly what the object looks like